src/frequenz/dispatch/_actor_dispatcher.py

```diff
@@ -179,10 +185,12 @@ async def _start_actor(self, dispatch: Dispatch) -> None:
             )
         else:
             _logger.info("Starting actor for dispatch type %r", dispatch.type)
-            self._actor = self._actor_factory(
+            actor = self._actor_factory(
                 dispatch_update, self._updates_channel.new_receiver()
```

Comment:
It still looks like this could benefit from `limit=1, warn_on_overflow=False`. See https://github.com/frequenz-floss/frequenz-dispatch-python/pull/104#discussion_r1933466664.